descriptors/ModuleDescriptor-template.json

```diff
@@ -370,6 +375,11 @@
       "displayName": "Circulation storage - anonymize loans for a user",
       "description": "Anonymize closed loans for a single user"
     },
+    {
+      "permissionName": "circulation-storage.anonymizeloans.post",
```

Comment:
please add the permission to `subPermissions` of `circulation-storage.all`

Reply:
Done.